docker/mysql/db/db_upgrade/portal/Update_RMC_DB_2024062000-2024071100.sql

```diff
@@ -0,0 +1,18 @@
+START TRANSACTION;
+
+ALTER TABLE `sites` ADD COLUMN `isVAPO` varchar(8) NOT NULL DEFAULT 'false' AFTER `decommissionTimestamp`;
+ALTER TABLE `sites` ADD INDEX `isVAPO` (`isVAPO`);
+ALTER TABLE `sites` ADD INDEX `site_type` (`site_type`);
```

Comment:
> ALTER TABLE `sites` ADD INDEX `site_type` (`site_type`);

This line prevents a successful database update because site_type already exists, caught in dev environment.